deploy-board/deploy_board/static/js/components/clusterconfigcomponents.js

```diff
@@ -388,6 +390,41 @@ Vue.component("placements-select", {
     }
 });
 
+Vue.component("hosttype-select", {
+    template: '<div v-bind:class="formStyle">\
+    <label class="deployToolTip control-label col-xs-2" data-toggle="tooltip" v-bind:title="title">{{label}}</label>\
+    <div v-bind:class="width"><div v-bind:class="groupStyle">\
+    <select v-bind:class="selectClass" v-on:change="updateValue($event.target.value)" required="true">\
+    <option v-for="option in selectoptions" v-bind:value="option.value" v-bind:selected="option.isSelected" v-bind:disabled="option.isDisabled">{{option.text}}</option></select>\
+    <span v-if="showhelp" class="input-group-btn">\
+      <button class="deployToolTip btn btn-default" type="button" data-toggle="tooltip" title="click to see more information" v-on:click="helpClick">\
+          <span class="glyphicon glyphicon-question-sign"></span>\
+      </button>\
+    </span></div>\
+    </div>\
+    </div>',
+    props: ['label', 'title', 'selectoptions', 'showhelp', 'small', 'selectsearch', 'retired'],
+    data: function () {
+        return {
+            width: this.small ? 'col-xs-4' : 'col-xs-10',
+            formStyle: this.small ? '' : 'form-group',
+            groupStyle: this.showhelp ? 'input-group' : '',
+            selectClass: this.selectsearch ? 'form-control single-select-search' : 'form-control'
+        }
+    },
+    methods: {
+        updateValue: function (value) {
+            this.$emit('input', value)
+        },
+        helpClick: function () {
+            this.$emit('helpclick')
+        },
+        retiredChange: function (value) {
+            this.$emit('retiredclick', value)
+        },
```

Comment:
Is this used? I tried searching for `retiredChange` / `retiredclick`, but no luck

Reply:
Not used. That was from an old commit. I removed it